docs/docs/winning-over-content-creators.md

```diff
@@ -0,0 +1,42 @@
+---
+
+Content creators are probably the heaviest end-users of your Gatsby projects because they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these.
+
+Some specific things that content creators care about include:
+
+- **Using tools they are familiar with.** The content creator probably has tools and workflows that they are already using, like a CMS. Gatsby could be used to integrate and possibly improve that workflow.
+- **Consistency** Consistency is key for a content creator, so their content matches the related content and branding.
+- **Audience**. Probably the biggest thing the content creator cares about is the audience, for who are they creating content?
+- **Flexible content**. Content creators care about personalizing content based on the audience that is viewing it, and creating multiple kinds of content.
+
+## Benefits
+
+Content creators may be curious about topics such as current workflow, new possibilities, flexibility.
+
+### Use the tools you already know and love
+
+> Gatsby can be configured to work with any CMS, database, or any other tools, including tools you're already using. If you love using your current CMS, great! Gatsby doesn’t decide your workflow, you do.
+
+### Choose the best tools for your content and audience
+
+> With Gatsby, you can pick whichever tools work best for your site and your content - choose the best CMS, the best ecommerce tool, the best video hosting, and build your own content stack without worrying about unreliable plugins or the politics of software partnerships. Have full control over how your content gets delivered to your audience.
+
+## Conclusion
+
+Gatsby offers unparalleled flexibility so you can have the benefits of Gatsby without compromising the efficiency and power of your favorite content tools. Content creators can keep their existing workflows and have plenty of room to grow their site in exciting new ways, empowering them to do what they do best, create captivating content for their audience.
+
+## Other resources
+
+- If your content creator is also a marketer [The winning over marketers](/docs/winning-over-marketers) could also be useful.
+- If you want to read up on how to integrate Gatsby with diffrent CMSs [This part of the docs is a great starting point](/docs/headless-cms). Another great article is ["Building Sites with Headless CMSs"](/blog/2018-2-3-sites-with-headless-cms/) by Shannon Soper.
+- Another great resource is the two-part blog by Michelle Gienow ["Delicious Simplicity ❤️ Gatsby: Building Happily Ever After"](/blog/2019-06-08-delicious-simplicity-case-study-part-1/) and ["Sharing the Gatsby Love with Content Creators"](/blog/2019-09-18-delicious-simplicity-case-study-part2/) by Linda Watkins.
+
+--
```

Comment:
```suggestion
```
As noted above, we can remove the stub status from this one with adequate content.